client/layout/masterbar/logged-in.jsx

```diff
@@ -207,38 +207,41 @@ class MasterbarLoggedIn extends Component {
 		return 'my-sites';
 	};
 
-	// will render as back button on mobile and in editor
-	renderMySites() {
-		const {
-			domainOnlySite,
-			hasMoreThanOneSite,
-			siteSlug,
-			translate,
-			isCustomerHomeEnabled,
-			section,
-		} = this.props;
-		const { isMenuOpen, isResponsiveMenu } = this.state;
+	renderWordPressIcon() {
+		const { siteSlug, translate, isCustomerHomeEnabled } = this.props;
+		const { isMenuOpen } = this.state;
 
 		const homeUrl = isCustomerHomeEnabled
 			? `/home/${ siteSlug }`
 			: getStatsPathForTab( 'day', siteSlug );
 
-		let mySitesUrl = domainOnlySite ? domainManagementList( siteSlug ) : homeUrl;
-		if ( 'sites' === section && isResponsiveMenu ) {
-			mySitesUrl = '';
-		}
-		const icon =
-			this.state.isMobile && this.props.isInEditor ? 'chevron-left' : this.wordpressIcon();
 		return (
 			<Item
-				url={ mySitesUrl }
+				url={ homeUrl }
+				icon={ this.wordpressIcon() }
+				isActive={ this.isActive( 'sites' ) && ! isMenuOpen }
+				tooltip={ translate( 'View a list of your sites and access their dashboards' ) }
```

Comment:
We need to change the tooltip descriptions. Maybe we can dynamically use the current site name
`View ${siteName} dashboard`